lib/app/utils/fhirpath_evaluator.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+require 'json'
+require 'fhir_models'
+
+module Inferno
+  class FHIRPathEvaluator
+    # Returns a new FHIRPathEvaluator if not testing and the external evaluator is selected.
+    # @param selected_evaluator [String] either 'internal' or 'external'
+    # @param external_evaluator_url [String] the base url for the external evaluator
+    def self.new_evaluator(selected_evaluator, external_evaluator_url)
+      return nil unless ENV['RACK_ENV'] != 'test' && selected_evaluator == 'external'
+
+      Inferno::FHIRPathEvaluator.new(external_evaluator_url)
+    end
+
+    # @param fhirpath_url [String] the base url for the FHIRPath /evaluate endpoint
+    def initialize(fhirpath_url)
```

Comment:
Looking over this branch is exposing some issues that are also present in the validator implementation. This class (as well as both of the validator classes) has no non-global state, so it could just be a module instead. This isn't necessarily something that needs fixing, just something that jumped out at me.